src/App.jsx

```diff
@@ -1,13 +1,240 @@
 
 import './App.css'
+import {useState} from "react";
+
+const CounterButton = ({ onIncrease, onDecrease }) => {
```

Comment:
De componenten an sich zien er goed uit, maar maak voortaan een mapje componenten aan en plaats ze in dit mapje. Als je altijd alle componenten in jouw App.jsx zou maken dan wordt die file super groot en daardoor onoverzichtelijk.